team-norms/delivery_healthcheck.md

```diff
@@ -6,15 +6,27 @@ It is the responsibility of the lead on each delivery team to provide an update
 
 A Delivery Healthcheck should answer:
 
-## Current iteration
+## Where you're working in defined iterations
+
+### Current iteration
  - When was your current iteration started?
  - What is your current iteration goal?
- - When is your next showcase?
- - Are you on track to meet customer expectations?
 
-## Next iteration
+### Next iteration
  - What is the goal for your next iteration?
 
-## Retrospective
+## Where you're working in a kanban flow
```

Comment:
I'm wondering whether `defined iterations` vs `kanban flow` is the right wording here.

On one hand you have iterations which produce a released increment of working software, that the customer can use *now* **end-to-end** versus what we're seeing which is a released increment of working software that works within a particular boundary e.g. API exposed on microservice X, UI built against design against stubbed API, etc. 

I think that it's important to not use the wrong wording because I see us as working in a kanban-based iterative workflow on all our deliveries.

Reply:
Major business goal stays the same over the long period of time and we define everything you listed for it: a timeframe (giant roadmap in Excel) and a collection of capabilities (Trello).

Then we pull minor goals and tasks according to whatever is happening - what is blocking us, what 3rd party is available for meeting with us, when is the next showcase at which business people require some update so they can fight for a budget and so on.

So kinda 1 and 2 together IMO.

Reply:
I feel similarly to Jakub - we're not cleanly in one of these two boxes. 

We do work towards a short-term goal (generally, "what do we want to show at the next showcase"?), and I think that's a healthy pressure for us.

We don't fully define the whole iteration though, rather pulling in small tasks one at a time from the backlog. You could argue that freedom within the iteration removes that "good pressure". We've had good feedback from showcases, though.